### DISCUSSION TITLE
Can you expand the loading animation effect on the basis of the button?

### DISCUSSION THREAD

**Discussion by junmaqiang** [fyne-io/fyne#4002] [Q&A]
## The effect is as follows

![截屏2023-06-25 13 16 49](https://github.com/fyne-io/fyne/assets/1819457/c1cd5228-21aa-433f-8fe0-7208359f2d3e)

## demo
https://www.iviewui.com/view-ui-plus/component/other/spin

**Comment by andydotxyz** [comment]
You can make any animation in your app https://developer.fyne.io/canvas/animation.
